e2e-tests/csv-source/upload.hydra

```diff
@@ -57,18 +88,21 @@ With Class cc:CSVSource {
     Expect Operation cc:GetCSVSourceAction {
         Invoke {
             Accept "text/csv"
-
         } => {
             Expect Status 200
-            Expect Header Content-Type 'text/csv; charset=utf-8' 
+            Expect Header Content-Type 'text/csv; charset=utf-8'
         }
 
-        Invoke { 
+        Invoke {
             Accept "application/ld+json"
         } => {
             Expect Status 200
             Expect Header Content-Type "application/ld+json; charset=utf-8"
             Expect Type <https://cube-creator.zazuko.com/vocab#CSVSource>
+            Expect Property csvw:quoteChar
+            Expect Property csvw:delimiter
+            Expect Property csvw:header
+            Expect Property csvw:headerRowCount
```

Comment:
@tpluscode Can we omit the Expect Property csvw:dialect here?

Reply:
No, these lines will always fail.